src/Slim/Middleware/Session.php

```diff
@@ -43,6 +43,7 @@ public function __construct($settings = [])
             'httponly'    => false,
             'name'        => 'slim_session',
             'autorefresh' => false,
+            'csrf'        => false
```

Comment:
This could mean that `slim/session` supports CSRF protection for Slim, which is not true. Also, why not to check for the presence of `slim/csrf` instead?
Tip: `class_exists()` can help. 😄